napari/utils/settings/_defaults.py

```diff
@@ -94,7 +115,7 @@ class Config:
 
     class NapariConfig:
         # Napari specific configuration
-        preferences_exclude = []
```

Comment:
let's add a class docstring to `PluginSettings` too ... otherwise it appears to be using the pydantic BaseSettings docstring for a tooltip, which looks funny :)

<img width="647" alt="p" src="https://user-images.githubusercontent.com/1609449/110861148-b98ca780-828b-11eb-8dac-34be013ee134.png">

Reply:
oh good catch!  will do that.  :)